src/app/getting-started/getting-started.component.html

```diff
@@ -113,16 +114,18 @@ <h2 class="circle">1</h2>
             </ng-template>
             <ng-template #osoUnlinked>
               <div class="checkbox">
-                <input type="checkbox" id="authOpenShift" name="authOpenShift"
-                       [(ngModel)]="authOpenShift">
+                <input type="checkbox" id="authOpenShift"
+                       [(ngModel)]="authOpenShift" [value]="openShiftURL">
                 <label for="authOpenShift" class="required-pf">OpenShift.com</label>
               </div>
             </ng-template>
           </div>
           <div class="form-group">
-            <button class="btn btn-lg btn-default" id="connect"
-                    [disabled]="isConnectAccountsDisabled"
-                    (click)="connectAccounts()">Connect accounts</button>
+              <input type="hidden"  name="token" [(ngModel)]="accessToken"/>
+              <input type="hidden"  name="redirect" [(ngModel)]="redirect"/>
```

Comment:
Should we be hiding an access token in plain text?

Reply:
Umm. yeah let me see if i can avoid this.. maybe fill/bind the input field just before the form actually gets submitted.